libraries/plugins/blockchain_history/devcommittee_history_api.cpp

```diff
@@ -0,0 +1,155 @@
+#include <scorum/blockchain_history/devcommittee_history_api.hpp>
+#include <scorum/blockchain_history/blockchain_history_plugin.hpp>
+#include <scorum/blockchain_history/schema/history_object.hpp>
+#include <scorum/app/api_context.hpp>
+#include <scorum/app/application.hpp>
+#include <scorum/blockchain_history/schema/operation_objects.hpp>
+#include <scorum/common_api/config.hpp>
+#include <scorum/protocol/operations.hpp>
+
+#include <map>
+
+namespace scorum {
+namespace blockchain_history {
+
+namespace detail {
+class devcommittee_history_api_impl
+{
+public:
+    scorum::app::application& _app;
+
+public:
+    devcommittee_history_api_impl(scorum::app::application& app)
+        : _app(app)
+    {
+    }
+
+    template <typename history_object_type, typename fill_result_functor>
+    void get_history(uint64_t from, uint32_t limit, fill_result_functor& funct) const
```

Comment:
NOTE: 
Different behaviour (with old interface of pagination). Don't forget to conform with front-end developers